politeiawww/backend.go

```diff
@@ -1977,7 +1977,7 @@ func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, user
 				propDetails.Token, err)
 		}
 		specVersion, err := strconv.ParseUint(propDetails.Version, 10, 64)
-		if err != nil || specVersion > latestVersion {
+		if err != nil || specVersion > latestVersion || specVersion == 0 {
```

Comment:
could you create specific errors for both version greater than latest and version = 0?

Reply:
Why?